spec/Pages/logIn_page.rb

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
`[STYLE]` The directory structure should also be all `snake_case`. The directory should be called `pages` in lowercase and `logIn_page` can be renamed to `log_in_page`.